code/style.css

```diff
@@ -0,0 +1,131 @@
+@import url("https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,400;0,600;0,700;0,800;1,100;1,400;1,600;1,700;1,800&display=swap");
+
+* {
+  box-sizing: border-box;
+  font-family: "Raleway", sans-serif;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+  gap: 5px;
+}
+
+header {
+  grid-column: span 8;
+  text-align: center;
+  background-color: #84a59d;
+  color: #fefae0;
+  font-size: 3vh;
+  padding: 15px;
+}
+
+header h2 {
+  font-style: italic;
+  font-weight: 400px;
+  font-size: 2vh;
+}
+
+main {
+  grid-column: span 8;
+  min-height: 60vh;
+  padding: 15px;
+  text-align: center;
+  background: #f7ede2;
+}
+
+.greeting-message {
+  grid-column-start: 2;
+  grid-column-end: 6;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  background: #f6bd60;
+  padding: 5vh;
+}
+
+.name {
+  margin: 10px, 0px;
```

Comment:
This line of code is not going to work, there shouldn't be any `,`. Moreover, if value is `0px`, we can omit `px` and just write `0`